crates/turborepo-signals/src/lib.rs

```diff
@@ -130,15 +144,25 @@ impl HandlerState {
 mod test {
     use std::{assert_matches::assert_matches, time::Duration};
 
+    use futures::stream;
+
     use super::*;
 
+    #[cfg(windows)]
+    const DEFAULT_SIGNAL: Signal = Signal::CtrlC;
+    #[cfg(not(windows))]
+    const DEFAULT_SIGNAL: Signal = Signal::Interrupt;
```

Comment:
Nit, but we could do and avoid the conditional compilation
```suggestion
    const DEFAULT_SIGNAL: Signal = if cfg!(windows) { Signal::CtrlC } else {Signal::Interrupt };
```

Reply:
I'm confused. Isn't this equivalent to your existing code, but just using a `cfg!` macro instead of the `cfg` directive?